docker-compose.yml

```diff
@@ -58,10 +62,14 @@ services:
 #      - postgres
 
   web:
-    build: .
+    build:
+      context: .
+      args:
+        UID: $UID
+        GID: $GID
     command: ["/source/tools/docker/web.sh"]
     ports:
-      - "80:80"
+      - "80:8080"
```

Comment:
Never mind the previous comment, I still wonder why you change the inner port, though.

Reply:
Thx